lmnet/tests/lmnet_tests/networks_tests/object_detection_tests/test_yolo_v1.py

```diff
@@ -231,7 +231,6 @@ def test_convert_boxes_space_inverse():
                 [
                     [50, 100, 400, 200, ],
                 ],
-
```

Comment:
Please avoid unnecessary file change :-1:

Reply:
fixed it